web-client/src/app/state/session-xrpl.service.ts

```diff
@@ -119,6 +132,37 @@ export class SessionXrplService {
     return txResponse;
   }
 
+
+  // Prepare transaction to autoload a new wallet with funds
+  async sendAutoFunds(
+    receiverId: string,
+    amount: number
+  ): Promise<xrpl.TxResponse> {
+
+    const public_key_hex = environment.xrpPublicKey;
+    const wallet_id = environment.xrpIssuer;
+    const pin = environment.autofundAccountPin;
+    // const pin = process.env.ISSUER_ENCLAVE_ACCESS;
+
+    const autoFundAmount = assetAmountXrp(amount);
+    const autoFundAmountXrp = convertFromAssetAmountXrpToLedger(autoFundAmount);
+
+    const preparedTx: xrpl.Payment = await this.prepareUnsignedTransaction(
+      receiverId,
+      autoFundAmountXrp,
+      environment.xrpIssuer
```

Comment:
can use `wallet_id` here

Reply:
in place of receiverId? in this scope wallet_id points to the issuer. I'm changing the name of that variable to issuer_id to avoid confusion.

Reply:
I meant we can use `issuer_id` const for line 147.